babel.config.json

```diff
@@ -2,7 +2,8 @@
     "presets": ["@babel/preset-env", "@babel/preset-react"],
     "plugins": [
         ["@babel/plugin-proposal-decorators", {"legacy": true}],
-        "@babel/plugin-transform-flow-strip-types"
+        "@babel/plugin-transform-flow-strip-types",
+        "@babel/plugin-proposal-nullish-coalescing-operator"
```

Comment:
TODO: adjust this in the `babel.config.json` of the `sulu/skeleton` too

Reply:
we would not need to do this if the `react-leaflet` package releases a new version that does not use the nullish coalescing operator. i will create an issue that explains the situation there. would propose to set this pull request on hold until that issue is either resolved or rejected 🙂

Reply:
@nnatter Looks like we will not get any feedback from @PaulLeCam. Looks like we need to hack this into here or what you think?

Reply:
yes i think so too 😢 
will add a pull request for the skeleton

Reply:
Have created a PR here: https://github.com/sulu/skeleton/pull/128